src/pydisagg/ihme/validator.py

```diff
@@ -38,9 +40,15 @@ def validate_interval(
         df.query(f"{lwr} >= {upr}")[index]
     ).to_list()
     if invalid_index:
-        raise ValueError(
-            f"{name} has invalid interval with index: {invalid_index}"
-        )
+        error_message = f"Index columns: ({', '.join(index)})\n"
+        if len(invalid_index) == 1:
+            error_message += f"{name} has invalid interval with 1 index: "
+        else:
+            error_message += f"{name} has invalid interval with {len(invalid_index)} indices: "
+        error_message += ", ".join(str(idx) for idx in invalid_index[:5])
+        if len(invalid_index) > 5:
+            error_message += " (showing first 5)"
+        raise ValueError(error_message)
```

Comment:
Looks great, minor suggestions:
* I don't think we need to distinguish when number of invalid rows is one or multiple, we can simplify a bit here
* We might want start a new line before and after we print the example indices to make the messages more readable